projects/packages/sync/src/class-lock.php

```diff
@@ -65,8 +84,18 @@ public function attempt( $name, $expiry = self::LOCK_TRANSIENT_EXPIRY ) {
 	 * @param bool|float $lock_expiration lock expiration.
 	 */
 	public function remove( $name, $lock_expiration = false ) {
-		if ( true === $lock_expiration || (string) get_option( self::LOCK_PREFIX . $name ) === (string) $lock_expiration ) {
-			delete_option( self::LOCK_PREFIX . $name );
+		global $wpdb;
+
+		$name       = self::LOCK_PREFIX . $name;
```

Comment:
these variable names would read better with lock_ as a prefix on all of them

Reply:
:thumbsup: